frontend/javascripts/admin/admin_rest_api.ts

```diff
@@ -949,42 +949,48 @@ export function getNewestVersionForTracing(
   );
 }
 
+export function hasSegmentIndexInDataStore(
+  dataStoreUrl: string,
+  dataSetName: string,
+  dataLayerName: string,
+  organizationName: string,
+) {
+  return doWithToken((token) =>
+    Request.receiveJSON(
+      `${dataStoreUrl}/data/datasets/${organizationName}/${dataSetName}/layers/${dataLayerName}/hasSegmentIndex?token=${token}`,
+    ),
+  );
+}
+
 export function getSegmentVolumes(
   requestUrl: string,
   mag: Vector3,
   segmentIds: Array<number>,
   additionalCoordinates: AdditionalCoordinate[] | undefined | null,
+  mappingName: string | null | undefined,
 ): Promise<number[]> {
+  let mappingNameNoEmptyString = mappingName;
+  if (mappingName != null && mappingName.length < 1) mappingNameNoEmptyString = null;
```

Comment:
```suggestion
  const mappingNameNoEmptyString = mappingName != null && mappingName.length < 1 ? null : mappingName;
```

do you know in which case the mapping name was an empty string? I would have thought that this should not happen :thinking:

Reply:
no, I don't. I can leave it out if we can rely on it, @fm3 just told me that it could cause problems if it was an empty string

Reply:
I think an empty string should not happen. if it does, it's a bug we should fix :) so, let's assume for now that an empty string won't happen.